content/news/release-20151224.md

```diff
@@ -0,0 +1,61 @@
+<script>
+ (function(d, t) {
+    var g = d.createElement(t),
+        s = d.getElementsByTagName(t)[0];
+    g.src = 'http://assets.gfycat.com/js/gfyajax-0.517d.js';
+    s.parentNode.insertBefore(g, s);
+}(document, 'script'));
+</script>
+
+We are proud to announce the Christmas 2015 release of OpenRA!
+
+<div style="text-align:center" markdown="1">
+<img class="gfyitem" data-id="PastDelightfulIberianbarbel" style="width: 600px; height: 300px"/>
+</div>
+40 developer elves have toiled long and hard to fill your stocking with lots of fixes, improvements and long-awaited features.
+
+In particular, this release re-introduces the global chat channel that is shared across all mods, which will allow you to meet up with other waiting players and arrange matches without joining servers. Going along with it is a re-design of the ingame server browser as well as the skirmish lobby.
```

Comment:
I don't think the global chat ever made it into a release, so the `re-` doesen't make sense here.